cla_backend/apps/cla_auth/views.py

```diff
@@ -16,6 +26,10 @@
 class AccessTokenView(Oauth2AccessTokenView):
     throttle_classes = [LoginRateThrottle]
 
+    def __init__(self, *args, **kwargs):
+        super(AccessTokenView, self).__init__(*args, **kwargs)
+        self.account_lockedout = False
```

Comment:
What is the purpose of this `self.account_lockedout` variable?
From what I can see it is set in [check_login_attempts](https://github.com/ministryofjustice/cla_backend/pull/850/files#diff-d415eef3965a9884491996b654d9f2a87f0dd5c732c8658d7c17b5b4a8de71e0R156) but that throws an exception and stops execution and doesn't continuing doing any further checks when attempts are more than settings.LOGIN_FAILURE_LIMIT